stubs/JACK-Client/jack/__init__.pyi

```diff
@@ -50,6 +50,8 @@ class Client:
         servername: str | None = ...,
         session_id: str | None = ...,
     ) -> None: ...
+    def __enter__(self) -> "Client": ...
+    def __exit__(self, *args: Any) -> None: ...
```

Comment:
Nit: typeshed generally uses `Any` for "can't be represented with the typing features that currently exist", and `object` for "accepts anything whatsoever".

```suggestion
    def __exit__(self, *args: object) -> None: ...
```

Reply:
Didn’t know that, thanks!